### ISSUE TITLE
loss hard to decrease

### ISSUE THREAD

**Issue by renrenzsbbb** [tencent-ailab/IP-Adapter#253]
Thanks for your great work. The result is surprising. So, I try to train or finetune your model using danbooru2000 dataset. Unfortunately, I find that the loss hardly decrease no matter train from scratch or finetune your model. I train it with batch size 16 in 2 * V100 GPUs. And then, I add a controlnet, the loss is also hard to decrease. Can you tell me your loss trend of change and give me some advices? Below is my tensorboard. Thanks in advance.
![tensorboard](https://github.com/tencent-ailab/IP-Adapter/assets/30514838/0071e12e-475d-44ba-9a3f-a0d92bf9c1f7)

**Comment by xiaohu2015** [comment]
I  think you should generate some visual images in your training Because loss is not a reliable indicator
